### ISSUE TITLE
Typo in configurable fields

### ISSUE THREAD

**Issue by cbandy** [kubernetes/minikube#5516]
`default-storageclass` is listed twice in `/cmd/minikube/cmd/config/config.go`. Functionality is unaffected because the definitions are identical.

https://github.com/kubernetes/minikube/blob/7d59b0d36cf5d0191095e41b859062dad90e3b11/cmd/minikube/cmd/config/config.go#L156-L161

https://github.com/kubernetes/minikube/blob/7d59b0d36cf5d0191095e41b859062dad90e3b11/cmd/minikube/cmd/config/config.go#L204-L209

**The exact command to reproduce the issue**:
```
curl -s 'https://raw.githubusercontent.com/kubernetes/minikube/master/cmd/minikube/cmd/config/config.go' \
| grep 'name:' | sort | uniq -c | awk '$1 != 1'
```
```
   2            name:        "default-storageclass",
```

**Comment by medyagh** [comment]
Ah thanks for cathing it !

I would be happy to review a PR if you decide to fix up the duplicate code

**Comment by tstromberg** [comment]
Fixed in v1.5.
